controller/status_test.go

```diff
@@ -6,9 +6,61 @@ import (
 	"reflect"
 	"testing"
 
+	. "github.com/onsi/gomega"
+
 	"github.com/giantswarm/formica/fleet"
 )
 
+func givenSingleUnitStatus(name, sliceID string) fleet.UnitStatus {
```

Comment:
Where is this naming coming from? I notice you prefix your helper methods with `given`.